styx-scheduler-service/src/test/java/com/spotify/styx/flyte/FlyteAdminClientRunnerTest.java

```diff
@@ -397,7 +395,7 @@ public void testPollingException() {
         .when(flyteAdminClient).getExecution(anyString(), anyString(), anyString());
 
     assertThrows(
-        FlyteRunner.PollingException.class,
+        RuntimeException.class,
```

Comment:
This test case is weird. Do we really expect this is a case? This is like testing any arbitrary Java method. :)